python/grass/jupyter/testsuite/display_test.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python
+
+############################################################################
+#
+# NAME:      display_test
+#
+# AUTHOR:    Caitlin Haedrich (caitlin dot haedrich gmail com)
+#
+# PURPOSE:   This is a test script for grass.jupyter's display module
+#
+# COPYRIGHT: (C) 2021 by Caitlin Haedrich and the GRASS Development Team
+#
+#            This program is free software under the GNU General Public
+#            License (>=v2). Read the file COPYING that comes with GRASS
+#            for details.
+#
+#############################################################################
+
+# Dependencies
```

Comment:
No need for a comment (would be good for an example code for learning Python, but not code which will be maintained for people who went at least though the initial hurdles of getting to know Python).